commerce-theme-speedwell/commerce-theme-speedwell-site-initializer/.gitignore

```diff
@@ -0,0 +1,3 @@
+out
+build
+classes
```

Comment:
I don't think we should need to add this file.... AFAIK, `build` and `classes` should already be ignored, but maybe `out` isn't? What's writing to `out`?

Reply:
@ethanbustad I saw many `out` directory being listed and available for git staging... I believe it's FE stuff, but didn't make sure, I was just remembering that, by the book, we have to git-ignore that directory.

Reply:
@gianmarcobrunialti alright, we can leave it in for now. There are some tests that ensure the files we include match what's expected, and we can just wait and see if there are any problems stemming from this change, and only fix when necessary.